plugin.yml

```diff
@@ -1,12 +1,13 @@
 name: Slapper
 author: jojoe77777
-version: 1.7.0
+version: 1.7.1
```

Comment:
Since the previous version bump you made was never merged into master, I think there is no need for another one